nntrainer/layers/activation_layer.h

```diff
@@ -155,6 +155,8 @@ class ActivationLayer : public Layer {
   std::function<Tensor(Tensor const &, Tensor &)> _act_fn;
   std::function<Tensor(Tensor const &, Tensor &, Tensor const &)> _act_prime_fn;
 
+  Tensor tempTensor;
```

Comment:
lets rename this with some proper name and description.

Reply:
done.